gui/pregnancy.lua

```diff
@@ -42,47 +56,82 @@ function PregnancyGui:init()
                 },
                 widgets.HotkeyLabel{
                     frame={l=0},
-                    label="Select Father",
+                    label="Set father to selected unit",
                     key='CUSTOM_SHIFT_F',
                     on_activate=self:callback('selectfather'),
                 },
                 widgets.HotkeyLabel{
                     frame={l=5},
-                    label="Set Mother's spouse as the Father",
+                    label="Set mother's spouse as the father",
                     key='CUSTOM_F',
                     on_activate=self:callback('spouseFather'),
                     disabled=function() return not self.mother or self.mother.relationship_ids.Spouse == -1 end
                 },
             },
         },
-        widgets.ResizingPanel{
-            frame={t=12},
+        widgets.Panel{
+            frame={t=12,h=14},
             frame_style=gui.FRAME_INTERIOR,
-            autoarrange_subviews=1,
             subviews={
                 widgets.HotkeyLabel{
-                    frame={l=0},
+                    frame={l=0, t=0},
                     key='CUSTOM_SHIFT_P',
                     label="Create pregnancy",
                     on_activate=self:callback('CreatePregnancy'),
                     enabled=function() return self.mother or self.father and self.father_historical end
                 },
-                widgets.TooltipLabel{
-                    text_to_wrap=self.msg,
-                    show_tooltip=true
-                },
-
                 widgets.ToggleHotkeyLabel{
+                    frame={l=1, t=1},
                     view_id='Force',
-                    label='Force',
+                    label='Replace existing pregnancy',
                     options={{label='On', value=true, pen=COLOR_GREEN},
                     {label='Off', value=false, pen=COLOR_RED}},
                     initial_option=false
                 },
+                widgets.TooltipLabel{
```

Comment:
Can you use a plain `Label` here? The attributes you're setting seem to undo everything that makes it a `TooltipLabel`